src/components/auth.tsx

```diff
@@ -25,9 +36,9 @@ export const Protect: React.FC<ProtectProps> = ({ role, orgSlug, children }) =>
 
     if (!isLoaded) return null
 
-    if (role == 'admin' && org.isAdmin) return children
-    if (role == 'researcher' && org.isResearcher) return children
-    if (role == 'reviewer' && org.isReviewer) return children
+    if (role == AuthRole.Admin && org.isAdmin) return children
+    if (role == AuthRole.Researcher && org.isResearcher) return children
+    if (role == AuthRole.Reviewer && org.isReviewer) return children
 
```

Comment:
why'd this file get updated? I don't see any changes, but am probably missing them.  it does seem like maybe good cleanups tho